python/pyspark/mllib/clustering.py

```diff
@@ -383,11 +383,11 @@ class GaussianMixtureModel(JavaModelWrapper, JavaSaveable, JavaLoader):
     >>> model.predict([-0.1,-0.05])
     0
     >>> softPredicted = model.predictSoft([-0.1,-0.05])
```

Comment:
For instance, weights within Gaussian mixture model:

JDK 8

```
weights: WrappedArray(0.49520257460263445, 0.33813075873069875, 0.16666666666666685)
```

JDK 11

```
weights: WrappedArray(0.5000000000000001, 0.33333333333333326, 0.16666666666666666)
```

Reply:
Also probably OK for the same reason. The test was too specific.